src/plugin.ts

```
@@ -276,7 +284,7 @@ export class TwitterRSSService extends Service {
       return;
     }
 
-    const intervalMinutes = parseInt(process.env.RSS_UPDATE_INTERVAL || '30');
+    const intervalMinutes = parseInt(this.getConfig('RSS_UPDATE_INTERVAL', '30') as string);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)